.NET/Microsoft.Recognizers.Text.DateTime.Tests/English/TestMergedParser.cs

```diff
@@ -37,6 +37,7 @@ public void BasicTestWithTwoResults(string text, string type1, string type2)
         [TestMethod]
         public void TestMergedParse()
         {
+            BasicTest("Set an appointment for Easter", Constants.SYS_DATETIME_DATE);

```

Comment:
Is there a test to make sure DateTime.MinValue is being filtered out from response? Please add one if not.

Reply:
done